sopel/modules/seen.py

```diff
@@ -36,17 +37,18 @@ def seen(bot, trigger):
         tz = get_timezone(bot.db, bot.config, None, trigger.nick,
                           trigger.sender)
         saw = datetime.datetime.utcfromtimestamp(timestamp)
+        delta = seconds_to_human((trigger.time - saw).total_seconds())
         timestamp = format_time(bot.db, bot.config, tz, trigger.nick,
                                 trigger.sender, saw)
 
-        msg = "I last saw {} at {}".format(nick, timestamp)
+        msg = "I last saw " + nick
         if Identifier(channel) == trigger.sender:
             if action:
-                msg = msg + " in here, doing " + nick + " " + message
+                msg = msg + " in here " + delta + ", doing: " + nick + " " + message
```

Comment:
While we are at it, I'd say that a `.format` would look better to me:

```
msg = 'I last saw {nick} in here {since}, doing: {nick} {action}'.format(
    nick=nick,
    since=delta,
    action=message
)
```

Of course, this would require to change the other cases accordingly.